handbook/Best practices/TypeScript/Project-References.md

```diff
@@ -0,0 +1,166 @@
+# Project References
+
+## Table of contents
+
+1. [Why use Project References?](#why-use-project-referencesgi)
+1. [Migrating a codebase](#migrating-a-codebase)
+1. [Tips](#tips)
+1. [Caveats](#caveats)
+
+
+## Why use Project References?
+
+Project References provides built-in scalability for TypeScript. It does this by providing developers tools to separate their code into smaller blocks. This enforces logical guidelines across the codebase which also results in healthier code over time. 
+
+The logical guidelines enforced by Project References allow TypeScript tools to operate on smaller chunks of work at a time. This in turn improves responsiveness and tightens the feedback between the VSCode and the TypeScript Language Server. 
+
+
+## Migrating a codebase
+
+TypeScript's documentation and guidance on [Project References](https://www.typescriptlang.org/docs/handbook/project-references.html) are trivial to implement in new codebases. In larger codebases, leveraging Project References can be cumbersome. This section outlines our best practices on migrating a large codebase to gradually adopt Project References.
```

Comment:
We might actually want to give an example of how they might structure project references in a novel codebase.

Reply:
What would a 'novel codebase' look like?